src/include/ipxe/efi/IndustryStandard/PeImage.h

```diff
@@ -42,6 +42,8 @@ FILE_LICENCE ( BSD3 );
 #define IMAGE_FILE_MACHINE_X64             0x8664
 #define IMAGE_FILE_MACHINE_ARMTHUMB_MIXED  0x01c2
 #define IMAGE_FILE_MACHINE_ARM64           0xAA64
+#define IMAGE_FILE_MACHINE_LOONGARCH32     0x6232
+#define IMAGE_FILE_MACHINE_LOONGARCH64     0x6264
```

Comment:
All EFI header changes will be handled by updating to copies of the current headers from EDK2.  I will handle this separately prior to this PR.

Reply:
This is now done: I have today pushed a series of 9 commits that between them bring the imported headers up to date and include the LoongArch64 `ProcessorBind.h`.

All headers that you need within `include/ipxe/efi` should now already be present once you rebase onto current master.